02-express-tutorial/answers/controllers/people.js

```diff
@@ -0,0 +1,58 @@
+let { people } = require('../../data');
+
+const getPeople = (req, res) => {
+	res.status(200).json({ success: true, data: people });
+};
+
+const createPerson = (req, res) => {
+	const { name } = req.body;
+	!name
+		? res.status(400).json({ success: false, msg: 'please provide name value' })
+		: null;
+	res.status(201).json({ success: true, person: name });
+};
+
+const createPersonPostman = (req, res) => {
+	const { name } = req.body;
+	!name
+		? res.status(400).json({ success: false, msg: 'please provide name value' })
+		: null;
+	res.status(201).json({ success: true, data: [...people, name] });
+};
+
+const updatePerson = (req, res) => {
+	const { id } = req.params;
+	const { name } = req.body;
+	const person = people.find((person) => person.id === Number(id));
+	!person
+		? res.status(404).json({ success: false, msg: `no person with id ${id}` })
+		: null;
```

Comment:
Again, make sure to write with a `return` statement if you're calling `res.status` anywhere that isn't the last line of the function